core/src/lib/bsock_testfind.cc

```diff
@@ -0,0 +1,1001 @@
+/*
```

Comment:
Why don't you derive from BareosSocketTCP? Then you only have to implement what is different instead of copying all the file.

Reply:
totally right had that in my mind but forgot to do it